test/topology/test_nodetool.py

```diff
@@ -0,0 +1,208 @@
+import pytest
+import asyncio
+import subprocess
+from test.pylib.manager_client import ManagerClient
+from test.pylib.util import wait_for_first_completed
+from test.topology.conftest import skip_mode
+
+
+async def validate_status_operation(result, live_eps, down_eps, leaving, joining, zero_token_eps):
+    """Validates the nodetool status output"""
+
+    lines = result.split('\n')
+    i = 0
+    dc_line = lines[i]
+    assert dc_line.startswith("Datacenter:")
+    result_dc = dc_line.split()[1]
+    assert result_dc == "datacenter1"
+    dc_line_len = len(dc_line)
+
+    i += 1
+    assert lines[i] == "=" * dc_line_len
+
+    i += 1
+    assert lines[i] == "Status=Up/Down"
+
+    i += 1
+    assert lines[i] == "|/ State=Normal/Leaving/Joining/Moving"
+
+    i += 1
+    assert lines[i].split() == ["--", "Address", "Load", "Tokens", "Owns", "Host", "ID", "Rack"]
+
+    visited_eps = set()
+    for j in range(len(live_eps + down_eps)):
+        i += 1
+        assert lines[i] != ""
+        pieces = tuple(lines[i].split())
+        if len(pieces) == 8:
+            status_state, ep, load, load_unit, tokens, owns, host_id, rack = pieces
+        else:
+            status_state, ep, load, tokens, owns, host_id, rack = pieces
+
+        assert ep not in visited_eps
+        visited_eps.add(ep)
+
+        assert ep in (live_eps + down_eps)
+
+        assert status_state[0] == ('U' if ep in live_eps else 'D')
+
+        if ep in joining:
+            assert status_state[1] == 'J'
+        elif ep in leaving:
+            assert status_state[1] == 'L'
+        else:
+            assert status_state[1] == 'N'
+
+        if ep in zero_token_eps:
+            assert int(tokens) == 0
+        else:
+            assert int(tokens) != 0
```

Comment:
You can (and probably should) check the exact value.
It is defined in `make_scylla_conf` in test/pylib/scylla_cluster.py:
```python3
        'num_tokens': 16,
```

You could extract the `16` into a constant in scylla_cluster.py, and import the constant here, and verify that `int(tokens)` is equal to it.